core/service/pool.go

```diff
@@ -175,3 +176,22 @@ func (i *Instance) Proposal() market.ServiceProposal {
 func (i *Instance) State() State {
 	return i.state
 }
+
+type instanceSocket struct {
```

Comment:
Name doesn't reflect purpose of this structure

Reply:
how you will name it ?